presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/model/PicPhotoFrame.kt

```diff
@@ -0,0 +1,49 @@
+package com.mashup.gabbangzip.sharedalbum.presentation.ui.model
+
+import androidx.annotation.DrawableRes
+import com.mashup.gabbangzip.sharedalbum.presentation.R
+
+enum class PicPhotoFrame(
+    val keyword: String,
+    @DrawableRes val frameResId: Int,
+) {
+    SNOWMAN(
+        keyword = GroupKeyword.SCHOOL.name,
+        frameResId = R.drawable.frame_snowman,
+    ),
+    CLOVER(
+        keyword = GroupKeyword.LITTLE_MOIM.name,
+        frameResId = R.drawable.frame_clover,
+    ),
+    FLOWER(
+        keyword = GroupKeyword.HOBBY.name,
+        frameResId = R.drawable.frame_flower,
+    ),
+    GHOST(
+        keyword = GroupKeyword.COMPANY.name,
+        frameResId = R.drawable.frame_ghost,
+    ),
+    PLUS(
+        keyword = GroupKeyword.NETWORK.name,
+        frameResId = R.drawable.frame_plus,
+    ),
+    SEXY(
+        keyword = GroupKeyword.EXERCISE.name,
+        frameResId = R.drawable.frame_sexy,
+    ),
+    HAMBURGER(
+        keyword = GroupKeyword.CREW.name,
+        frameResId = R.drawable.frame_hamburger,
+    ),
+    ;
+
+    companion object {
+        fun getType(frameType: String): PicPhotoFrame {
+            return entries.associateBy { it.name }[frameType] ?: PLUS
+        }
+
+        fun getFrontType(keyword: String): PicPhotoFrame {
+            return entries.associateBy { it.keyword }[keyword] ?: PLUS
+        }
```

Comment:
```suggestion
        fun getTypeByName(typeName: String): PicPhotoFrame {
            return entries.associateBy { it.name }[typeName] ?: PLUS
        }

        fun getTypeByKeyword(keyword: String): PicPhotoFrame {
            return entries.associateBy { it.keyword }[keyword] ?: PLUS
        }
```
뭔가.. getType, getFrontType 을 쓰기엔 둘 다 무언가를 기반으로 PicPhotoFrame 을 가져오는 함수들이라 getTypeByxxx 형태로 이름을 지어보는 것은 어떨까 하는 생각이 들었는데 어떤것 같아...?!

Reply:
하.. 저거 네이밍 할 때 사실 엄청 불편하긴했어.. by~~ 이거 뭔가 괜찮은거 같아..! 이름 수정해서 올릴게~!

Reply:
반영완. https://github.com/mash-up-kr/gabbangzip-Android/pull/99/commits/d6bba4ad1673ec99afed201195900cbd3414c34f